netbout-web/src/main/js/inbox.js

```diff
@@ -51,48 +51,61 @@ $(document).ready(
               dataType: 'xml',
               method: 'GET',
               success: function (data) {
-                var appendix = '<ul class="bouts">';
+                var appendix = '<ul class="bouts">', $data = $(data),
```

Comment:
@erimerturk Let's avoid declaring multiple variables on the same line as it makes the code hard to read.